salt/metalk8s/salt/master/certs/salt-api.sls

```diff
@@ -0,0 +1,46 @@
+{%- from "metalk8s/map.jinja" import kube_api with context %}
+
+include:
+  - metalk8s.internal.m2crypto
+
+Create Salt API private key:
+  x509.private_key_managed:
+    - name: /etc/kubernetes/pki/salt-api.key
+    - bits: 2048
+    - verbose: False
+    - user: root
+    - group: root
+    - mode: 600
```

Comment:
Why not 400? even if that does not change a lot...

Reply:
Or is that because if we want to renew it we would need 600?

Reply:
Good question.
But since it's like that for every certs, if we want to change it we should probably change it for all certs